### PR TITLE
add OpenStack etcd-manager support

### PR DESCRIPTION
**Pull request by zetaab**
adds openstack support for etcd-manager, no need to use legacy anymore.

cc @drekle 

/hold

lets wait that openstack support for etcd-manager is added

### CODE REVIEW

**Comment by zetaab** [comment]
hold until https://github.com/kopeio/etcd-manager/pull/245 is merged

**Comment by justinsb** [comment]
I just uploaded etcd-manager 3.0.20190816 also, with the PRs for OpenStack & DigitalOcean incorporated.

**Comment by zetaab** [comment]
great!

**Comment by zetaab** [comment]
@justinsb I updated the etcd-manager in this PR as well. No need to do separate PR

**Comment by justinsb** [comment]
Thanks @zetaab 

/approve
/lgtm

File: pkg/model/openstackmodel/firewall.go [hunk 1]
```diff
@@ -442,7 +442,7 @@ func (b *FirewallModelBuilder) addProtokubeRules(c *fi.ModelBuilderContext, sgMa
 			Direction:    s(string(rules.DirIngress)),
 			Protocol:     s(string(rules.ProtocolTCP)),
 			EtherType:    s(string(rules.EtherType4)),
-			PortRangeMin: i(3998),
+			PortRangeMin: i(3994),
```
Comment (drekle) [inline pkg/model/openstackmodel/firewall.go]: If this is for etcd manager can we add a comment?  I've been trying to do this with self documenting code and the method names, however I think its probably fine here with a comment as its just taking some of the work out of protokube.  

Also not sure if there is a sane way of checking that we are using etcd manager and only opening the port in the event that the provider is not `Legacy`.

**Pull request merged**